prow/cmd/deck/tide.go

```diff
@@ -162,6 +163,9 @@ func (ta *tideAgent) filterHiddenPools(pools []tide.Pool) []tide.Pool {
 	filtered := make([]tide.Pool, 0, len(pools))
 	for _, pool := range pools {
 		needsHide := matches(pool.Org+"/"+pool.Repo, ta.hiddenRepos)
+		if needsHide && ta.showHidden {
+			filtered = append(filtered, pool)
+		}
 		if needsHide == ta.hiddenOnly {
```

Comment:
Can this be `else if`? As it stands I can imagine future logic changes causing surprise duplicate entries.

Same goes for the two below.

Reply:
Done